test/functional/ui/screen.lua

```diff
@@ -394,19 +461,39 @@ function backward_find_meaningful(tbl, from)
   return from
 end
 
-function get_attr_id(attr_ids, attrs)
+function get_attr_id(attr_ids, attr_ignore, attrs)
   if not attr_ids then
     return
   end
   for id, a in pairs(attr_ids) do
-    if a.bold == attrs.bold and a.standout == attrs.standout and
-       a.underline == attrs.underline and a.undercurl == attrs.undercurl and
-       a.italic == attrs.italic and a.reverse == attrs.reverse and
-       a.foreground == attrs.foreground and
-       a.background == attrs.background then
+    if equal_attrs(a, attrs) then
        return id
      end
   end
+  if attr_ignore == nil or attr_index(attr_ignore, attrs) ~= nil then
```

Comment:
@justinmk Yes, that would be better (or rather, attr_ignore should then default to `{{}}`, just ignoring the empty attr set, i e normal text), but when I tried to enable this before, the mouse tests hang, and it was not possible to debug in any obvious way, due to busted not printing _any_ information until all tests are done (it is not even possible to see which test it is waiting for...) and `luajit` not printing a backtrace or anything on `^C`. I will try again debugging this using pure lua, now that this is possible.